runbot_send_email/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+========================
+Runbot Send Email Result
+========================
+
+This addon sends the results of the Runbot builds through email, Runbot send the notification to all followers of build, the notification has the following items:
+
+* **General Info:** Contains the Status of build, Committer name, SHA Commit, Commit description. In this section runbot will indicate status changes of the build, showing color and icon related with such build status.
+
+ .. image:: https://raw.githubusercontent.com/Vauxoo/runbot-addons/11.0/runbot_send_email/static/img/gi.png
```

Comment:
Use figure instead of image and “./MODULE/static...” parh instead of url

Reply:
@moylop260, change done.